.github/workflows/test.yml

```diff
@@ -9,20 +9,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby_version: ["2.3.8", "2.4.5", "2.5.3", "2.6.3", "2.7.2"]
+        ruby_version: ["2.3", "2.4", "2.5", "2.6", "2.7", "3.0", "3.1"]
```

Comment:
Bump all Ruby patch versions to latest, and promote Ruby 3.0 and 3.1 to non-experimental status.